src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dao/UpdateDao.java

```diff
@@ -45,7 +46,34 @@ public class UpdateDao {
 
     @Field("review_trusts")
     private List<TrustDataToReviewDao> reviewTrusts;
+    @Field("ceased_to_be_registrable_beneficial_owner")
+    private RelevantStatementsType relevantStatementsType;
+    @Field("trust_involved_in_the_oe")
+    private RelevantStatementsType relevantStatementsType2;
+    @Field("become_or_ceased_beneficiary_of_a_trust")
+    private RelevantStatementsType relevantStatementsType3;
```

Comment:
Should these be the new names `ceased_to_be_registrable_beneficial_owner`, `trust_involved_in_the_oe`, `become_or_ceased_beneficiary_of_a_trust`